CMP1903_A1_2324/Game.cs

```diff
@@ -10,10 +12,55 @@ internal class Game
     {
         /*
          * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
+         
+         *
+         *
          *
          * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
+        //
+       
+        public Die Die1;
+        public Die Die2;
+        public Die Die3;
+        public int sum_of_dice;
+        
+        public Game() {
+
+         Random rng = new Random();
+        Die3 = new Die(rng);
+        Die2 = new Die(rng);
+        Die1 = new Die(rng);
```

Comment:
Not an actual complaint, it just bugs me that it goes 3, 2, 1 instead of 1, 2, 3.